doc/Requirements Specification/Parts/Discussion.lyx

```diff
@@ -357,21 +344,7 @@ glspl{component}
 
 \end_inset
 
- interact with each other.
- If a 
-\begin_inset ERT
-status open
-
-\begin_layout Plain Layout
-
-
-\backslash
-gls{component}
-\end_layout
-
-\end_inset
-
- calls another 
+ (or bundles) at a time, some internal 
```

Comment:
it’s unclear what “bundles” refers to here. How about changing the sentence to “As Beagle should be able to measure specific components” (you can omit “at a time”, I think)